### DISCUSSION TITLE
Rethinking of Blinds mod

### DISCUSSION THREAD

**Discussion by Givikap120** [ppy/osu#25322] [Ideas]
For now, Blinds looks very clunky to me. The idea of the mod is being pure memory is funny, but those wooden shutters on both sides it's not what blinds should like in my opinion.
Make it as NM, but with building combo, objects slowly become transparent, becoming fully invisible on X combo. When a player miss/SB - the transparency value becomes 2x lower (from 1.0 to 0.5 for example), and then by not missing it becomes invisible again. This way Blinds would really look smooth instead of clunky while keeping the original idea of the mod.

**Comment by WalmartWhiteCat** [comment]
But wouldn't that mean it's not Blinds anymore? That's just No Scope but for circles instead of the cursor, or am I wrong? I think Blinds works just fine, and making these kinds of changes would basically make it a different mod altogether. It's called Blinds because there are, well, blinds.

However, I do like the idea. It's cool and I think people would like it, but I feel it would be better as a separate mod instead of a rework of an already existing mod.

**Reply by Givikap120** [reply]
Nobody will play blinds with competition purpose if more smooth alternative will exist.
I'm completely fine with blinds being with other Fun mods like Mute, which will be played very rarely.
But if it would be the only "Full memory mod" - I don't like it

**Reply by WebKun** [reply]
It looks like "mute" mod, But is invisible hitobject version?
